Automattic/woocommerce-payments
# Initial Setup of Psalm for WC Pay Client

#### Changes proposed in this Pull Request









This PR sets up the config files to run Psalm within the client. Initial results look fairly promising in terms of the number of errors. There seem to be some related to Psalm not resolving any Subscriptions related classes which we can probably fix with adjustments to the config or by creating stubs, but I haven't looked into this yet.

#### Testing instructions





* `./vendor/bin/psalm` (or `npm run psalm`) and check the results.

-------------------
